backend/internal/config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
```

Comment:
You could add all config values in one struct. That'd let you easily plug in [viper](https://github.com/spf13/viper), or some env unmarshaler

Reply:
We don't think we need something like viper for now, but we will consider it for later

Reply:
Doing it in a struct would make it much easier, when you decide to do it